test/test_fcntl.rs

```diff
@@ -132,3 +133,15 @@ mod linux_android {
     }
 
 }
+
+#[test]
+fn test_fallocate() {
```

Comment:
This needs to be moved into the module above as I did, so it only runs on linux/android.

Reply:
Argh.. my bad. Just saw the build failures. Had missed that. Fixed it now. Thank you very much.